Makefile.pre.in

```diff
@@ -1577,9 +1583,11 @@ FROZEN_FILES_OUT = \
 		Python/frozen_modules/frozen_only.h
 # End FROZEN_FILES_OUT
 
-Programs/_freeze_module.o: Programs/_freeze_module.c Makefile
+Programs/_freeze_module.o: $(srcdir)/Programs/_freeze_module.c Makefile $(srcdir)/Include/Python.h $(srcdir)/Include/abstract.h $(srcdir)/Include/bltinmodule.h $(srcdir)/Include/boolobject.h $(srcdir)/Include/bytearrayobject.h $(srcdir)/Include/bytesobject.h $(srcdir)/Include/ceval.h $(srcdir)/Include/codecs.h $(srcdir)/Include/compile.h $(srcdir)/Include/complexobject.h  $(srcdir)/Include/critical_section.h $(srcdir)/Include/descrobject.h $(srcdir)/Include/dictobject.h $(srcdir)/Include/enumobject.h $(srcdir)/Include/exports.h $(srcdir)/Include/fileobject.h $(srcdir)/Include/fileutils.h $(srcdir)/Include/floatobject.h $(srcdir)/Include/genericaliasobject.h $(srcdir)/Include/import.h $(srcdir)/Include/intrcheck.h $(srcdir)/Include/iterobject.h $(srcdir)/Include/listobject.h $(srcdir)/Include/lock.h $(srcdir)/Include/longobject.h $(srcdir)/Include/marshal.h $(srcdir)/Include/memoryobject.h $(srcdir)/Include/methodobject.h $(srcdir)/Include/modsupport.h $(srcdir)/Include/moduleobject.h $(srcdir)/Include/monitoring.h $(srcdir)/Include/object.h $(srcdir)/Include/objimpl.h $(srcdir)/Include/osmodule.h $(srcdir)/Include/patchlevel.h $(srcdir)/Include/pyatomic.h $(srcdir)/Include/pybuffer.h $(srcdir)/Include/pycapsule.h $(srcdir)/Include/pyerrors.h $(srcdir)/Include/pyframe.h $(srcdir)/Include/pyhash.h $(srcdir)/Include/pylifecycle.h $(srcdir)/Include/pymacro.h $(srcdir)/Include/pymath.h $(srcdir)/Include/pymem.h $(srcdir)/Include/pyport.h $(srcdir)/Include/pystate.h $(srcdir)/Include/pystrcmp.h $(srcdir)/Include/pystrtod.h $(srcdir)/Include/pythonrun.h $(srcdir)/Include/pytypedefs.h $(srcdir)/Include/rangeobject.h $(srcdir)/Include/refcount.h $(srcdir)/Include/setobject.h $(srcdir)/Include/sliceobject.h $(srcdir)/Include/structseq.h $(srcdir)/Include/sysmodule.h $(srcdir)/Include/traceback.h $(srcdir)/Include/tupleobject.h $(srcdir)/Include/typeslots.h $(srcdir)/Include/unicodeobject.h $(srcdir)/Include/warnings.h \
```

Comment:
Wouldn't it be more aesthetically pleasing to arrange the dependencies from top to bottom?
```
$(srcdir)/Objects/stringlib/split.h \
$(srcdir)/Objects/stringlib/ucs1lib.h \
$(srcdir)/Objects/stringlib/ucs2lib.h \
$(srcdir)/Objects/stringlib/ucs4lib.h \
$(srcdir)/Objects/stringlib/undef.h \
```